docs/faq.md

```diff
@@ -351,29 +351,30 @@ If you run experiments with tabulated benchmarks using the `SimulatorBackend`, a
 
 ### <a name="remote-tuning"></a> I don’t want to wait, how can I launch the tuning on a remote machine?
 
-You can use the remote launcher to launch an experiment on a remote machine. The
-remote launcher supports both `LocalBackend` and `SageMakerBackend`. In the
-former case, multiple trials will be evaluated on the remote machine (one use-case
-being to use a beefy machine), in the latter case trials will be evaluated as
-separate SageMaker training jobs.
+Remote launching of experiments has a number of advantages:
+* The machine you are working on is not blocked
+* You can launch many experiments in parallel
+* You can launch experiments with any instance type you like, without having to
+  provision them yourselves
 
-Here is an example for how to run tuning with the remote launcher:
+Remote launching for local, simulator, and SageMaker back-end is detailed in
+[this tutorial](tutorials/benchmarking/README.md), see also these examples:
```

Comment:
This is mixing benchmarking with backend, and they are two separate concepts. It can confuse first time audiences. I suggest we add a tiny tutorial on depicting differences between various backends, link it here, and in addition to that link to `tutorials/benchmarking/README.md` for usage examples.

Reply:
OK, we need to write that additional tutorial (on back-ends).

Reply:
Unfortunately, remote launching *is* different for the different back-ends.